edx_django_utils/monitoring/code_owner/tests/test_middleware.py

```diff
@@ -12,7 +12,7 @@
 
 from edx_django_utils.monitoring.code_owner.middleware import CodeOwnerMonitoringMiddleware
 from edx_django_utils.monitoring.code_owner.tests.mock_views import MockViewTest
-from edx_django_utils.monitoring.code_owner.utils import _process_code_owner_mappings
+from edx_django_utils.monitoring.code_owner.utils import clear_cached_mappings
```

Comment:
**Reviewer Note**: Part of this PR simply provides a cleaner implementation of the mapping setup and testing. `clear_cached_mappings` is used in place of all the messy patches of `_PATH_TO_CODE_OWNER_MAPPINGS`.